src/components/Courses/Courses.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import CourseCard from './components/CourseCard/CourseCard';
+import SearchBar from './components/SearchBar/SearchBar';
+import Button from '../../common/Button/Button';
+
+export default function Courses({ course }) {
+	function showCreateCourse() {
+		document.getElementById('courses').style.display = 'none';
+		document.getElementById('createCoursePortal').style.display = 'block';
+	}
+
+	return (
+		<div id='courses'>
+			<div className='coursesBar'>
+				<SearchBar></SearchBar>
+				<Button buttonText='Add New Course' onClick={showCreateCourse}></Button>
+			</div>
+
+			{course?.map((course) => {
+				return (
+					<div style={({ border: '2px solid red' }, { margin: '5%' })}>
```

Comment:
1. Inline style is an evil. Please, use CSS classes always to make single entry point for your styles editing;
2. If you need to somehow manage you styles in React conditionally, please, pay attention for [classnames](https://www.npmjs.com/package/classnames) package;
3. In this situation you should just declare styles in one object;